internal/blobcache/blobcache.go

```diff
@@ -11,29 +11,29 @@ import (
 
 // Crude estimate of the overhead per blob: a SHA-256, a linked list node
 // and some pointers. See comment in blobCache.add.
-const cacheOverhead = len(restic.ID{}) + 64
+const CacheOverhead = len(restic.ID{}) + 64
```

Comment:
There's no need for this to be exported. The only external reference is in a comment, which can add the package name instead.

Reply:
Hm, it's also used in https://github.com/restic/restic/blob/bf9c8771a469f1ee0d4b6fab78791d36426371b2/internal/fuse/fuse_test.go#L30 (which admittedly accidentally wasn't updated in the PR you reviewed) so I'd rather do export it?

Reply:
I forgot about that one. I guess it will have to be exported then.